sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/config/data/Rule.java

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.owaspzapwrapper.config.data;
+
+public class Rule {
```

Comment:
It might be beneficial to make the objects immutable (no setters). Is there any reason, why the rule needs to change after it's creation?